content/news/053/index.md

```diff
@@ -121,6 +121,33 @@ _Check out the [web examples][chuot-examples]!_
 
 ## Library Updates
 
+### Godot Rapier: Rewrite it in Rust
+
+{{ video_figure(
+    type="video/mp4",
+    alt="Godot + Rapier logo make up the 'Godot Rapier' logo ; a few balls fall on it, showing off soft physics.",
+    src="godot_rapier.mp4",
+    caption="Godot Rapier showcase") }}
+
+[Godot Rapier][godot_rapier_github] is a Physics library for [Godot][godot], using [Rapier][rapier]:
+a physics engine written in Rust.
+
+Godot Rapier was written in C++, and is in the final "beta" stage of
+being rewritten completely in Rust.
+
+Its author wrote [an article on
+Reddit][godot_rapier_reddit] with more details. Since then,
+the web exports have been fixed.
```

Comment:
Read more about this on the `[Rapier Godot Progress](https://godot.rapier.rs/docs/progress)` page.